AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DisplayMapperAdded.py

```diff
@@ -98,20 +157,202 @@ def AtomEditorComponentsLevel_DisplayMapper_AddedToEntity():
             display_mapper_component.get_component_property_value(
                 AtomComponentProperties.display_mapper('LDR color Grading LUT')) == display_mapper_asset.id)
 
-        # 5. Set Enable LDR color grading LUT property True
+        for operation_type in DISPLAY_MAPPER_OPERATION_TYPE.keys():
+            # 5. Set the Display Mapper Operation Type enumerated in DISPLAY_MAPPER_OPERATION_TYPE for each type
+            # Type property cannot be set currently. as a workaround we are calling an ebus to set the operation type
+            # display_mapper_component.set_component_property_value(
+            #     AtomComponentProperties.display_mapper('Type'), DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            render.DisplayMapperComponentRequestBus(
+                bus.Broadcast, "SetDisplayMapperOperationType", DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            general.idle_wait_frames(3)
+            set_type = render.DisplayMapperComponentRequestBus(bus.Broadcast, "GetDisplayMapperOperationType")
+            Report.info(f"DiplayMapperOperationType: {set_type}")
```

Comment:
we aren't validating anything in this step 5 are we? if we're using a workaround to do something but not testing anything, shouldn't we just not add it until it can be added and validated as we normally do? maybe I am missing something here

Reply:
I can Report.result instead of info here. Once the workaround is replaced we would be verifying that the DisplayMapperOperationType is correctly set.